src/components/LoginPage/Login.js

```diff
@@ -3,35 +3,63 @@ import { Link } from 'react-router-dom';
 import { useNavigate } from 'react-router-dom';
 import { useAuth } from '../../context/AuthContext';
 import { loginService } from '../../utils/AuthFunctions';
+import { useState } from 'react';
 
 const Login = () => {
     const navigate = useNavigate();
     const { setInitialAuth  } = useAuth();
-    
+    const [loginData, setLoginData] = useState({
+        email:'',
+        password: ''
+    });
+
+    const loginInputHandler = (e) => {
+        setLoginData((prev) => ({...prev, [e.target.name]: e.target.value}))
+    }
+
+    const loginHandler = async () => {
+        const respData = await loginService(loginData.email, loginData.password)
+        if(respData){
+            setInitialAuth({isLoggedIn: true,
+                token: respData.data.encodedToken});
+            localStorage.setItem('token', respData.data.encodedToken)
+            localStorage.setItem('isLoggedIn', true)
+            navigate('/homePage') 
```

Comment:
i think there is no need to store the logged in state in local storage. If token is present in the local storage you can set the logged in state to true